celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -423,55 +121,68 @@ def runner(tech_matrix, F,i,l,j,k,route_id,final_demand_scaler):
         """
         tim0 = time.time()
         res = pd.DataFrame()
-        res2 = pd.DataFrame()
-        res,res2 = solver_optimization(tech_matrix, F)
+        res = solver(tech_matrix, F,process_emissions)
         res['value'] = res['value']*final_demand_scaler
         if not res.empty:
           res.loc[:,'year'] =  i
           res.loc[:,'facility_id'] =  l
           res.loc[:,'stage'] = j
           res.loc[:,'material'] = k
-          res.loc[:, 'route_id'] = route_id
+          res.loc[:,'route_id'] = route_id
     
-          print(str(i) +' - '+j + ' - ' + k)
+          print(str(i) +' - '+j + ' - ' + k, flush = True)
     
         else:                
-          print(f"optimization pylca-opt-background emission failed for {k} at {j} in {i}") 
-          pass
-       
-        if not res2.empty:
-          res2.loc[:,'year'] =  i
-          res2.loc[:,'facility_id'] =  l
-          res2.loc[:,'stage'] = j
-          res2.loc[:,'material'] = k
-          res2.loc[:, 'route_id'] = route_id
-    
-        else:
-           print(f"optimization pylca-opt-background emission failed for {k} at {j} in {i}")
-           pass
+          print(f"pylca-opt-background emission failed for {k} at {j} in {i}", flush = True) 
+          pass       
+        
     
         print(str(time.time() - tim0) + ' ' + 'taken to do this run',flush=True)
```

Comment:
Do we always need the timing printed out? Can there be a `verbose` parameter added that controls how much information is printed?

Reply:
A verbose parameter would be a good way to stop these print statements (also for the previous comment). One question, is there an already used verbose parameters in the code that I can pull from __main__ or scenario or should I create one within the LCA environment? @rjhanes

Reply:
Added verbose to LCA